templates/header/topbar_begin.html

```diff
@@ -9,8 +9,11 @@
 <div class="nav-container">
     <div class="container">
         <div class="pure-menu pure-menu-horizontal" role="navigation" aria-label="Main navigation">
-            <form action="/releases/search" method="GET"
-                class="landing-search-form-nav {% if is_latest_version is defined and not is_latest_version %}not-latest{% endif %}">
+            <form action="/releases/search"
+                  method="GET"
```

Comment:
Why did you put it on another line?

Reply:
When wrapping a tag, I generally prefer to place each attribute on their own line because it can be harder to visually locate the attribute otherwise. If this is something you'd rather avoid in the markup, I'm fine restoring it :+1:

Reply:
No personal preference here, just wondering. :)